projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/marker-tooltip/marker-tooltip.component.scss

```diff
@@ -0,0 +1,60 @@
+@import 'color-palette';
+@import 'font';
+
+.marker-tooltip-container {
+  @include body-small(white);
+  background-color: $gray-9;
+  box-shadow: 0px 4px 5px rgba(0, 0, 0, 0.2), 0px 3px 14px rgba(0, 0, 0, 0.12), 0px 8px 10px rgba(0, 0, 0, 0.14);
+  border-radius: 4px;
+  padding: 8px 12px;
+  margin: 8px;
+  width: 320px;
+  max-height: 172px;
+
+  .tooltip-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .log-count {
+      display: flex;
+      align-items: center;
+
+      .count {
+        width: 18px;
+        height: 18px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        background-color: $gray-5;
+        margin-left: 4px;
+        border-radius: 4px;
+      }
+    }
+  }
+
+  .divider {
+    margin: 4px 0;
+    border: 1px solid $gray-6;
+  }
+
+  .attribute {
+    display: flex;
+    align-items: center;
+    @include ellipsis-overflow();
```

Comment:
inclusions first in a css rule please. we wouldn't know what got overridden otherwise :)

Reply:
done